src/ImageGenerator.py

```diff
@@ -0,0 +1,95 @@
+import os
+import re
+import sys
+from typing import List
+
+from PIL import Image
+
+
+# mapファイルのテキストデータを2次元配列に変換するメソッド
+def changEArray(map_data: List[str]) -> List[List[str]]:
+    map_list = []
+    # 各行を一マスごとに分割して配列に格納
+    for i in range(len(map_data)):
+        if map_data[i] != "" and map_data[i][0] == "D":
+            map_list.append(re.split(",|\n", map_data[i].replace("D:", "")))
+        elif map_data[i] != "" and map_data[i][0] == "H":
+            hot = re.split(",|\n", map_data[i].replace("H:", ""))
+        elif map_data[i] != "" and map_data[i][0] == "C":
+            cool = re.split(",|\n", map_data[i].replace("C:", ""))
+
+    map_list[int(cool[1])][int(cool[0])] = "C"
+    map_list[int(hot[1])][int(hot[0])] = "H"
+
+    return map_list
+
+
+# 一つのmapファイルをpngに変換するメソッド
+def draWPng(file_name: str) -> None:
+    # mapファイルを開いてmap_dataにテキストを格納
+    # print(file_name)
+    file = open("../" + file_name, "r")
+    map_data = file.read().split("\n")
+    file.close
```

Comment:
ファイルを読むときは `with`を用いて書くとcloseを自動的にやってくれるので
安全です！
```suggestion
    with open("../" + file_name, "r") as file
        map_data = file.read().split("\n")
```